MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.Light.xaml

```diff
@@ -20,6 +20,11 @@
     <SolidColorBrush x:Key="MaterialDesignDivider" Color="#1F000000" po:Freeze="True" /> <!-- 12% -->
     <SolidColorBrush x:Key="MaterialDesignSelection" Color="#FFDeDeDe" po:Freeze="True" />
 
+    <SolidColorBrush x:Key="MaterialDesignToolButtonForeground" Color="#616161" po:Freeze="true" />
+    <SolidColorBrush x:Key="MaterialDesignPopupBoxForeground" Color="#616161" po:Freeze="true" />
+    <SolidColorBrush x:Key="MaterialDesignFlatDefaultToggleButtonForeground" Color="#616161" po:Freeze="true" />
+    <SolidColorBrush x:Key="MaterialDesignFlatDefaultToggleButtonBackground" Color="#e0e0e0" po:Freeze="true" />
```

Comment:
These resources should match the ones in MaterialDesignTheme.Dark.xaml

Reply:
Yeap, that means that it only applies to the button and popup box controls since the flat button makes use of the MaterialDesignToolBackground/MaterialDesignToolForeground.